kythe/proto/BUILD

```diff
@@ -3,6 +3,7 @@ load("@rules_java//java:defs.bzl", "java_proto_library")
 load("//:visibility.bzl", "PUBLIC_PROTO_VISIBILITY")
 load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 load(":go.bzl", "go_kythe_proto")
+load("@npm_bazel_labs//:index.bzl", "ts_proto_library")
```

Comment:
There's also https://github.com/Dig-Doug/rules_typescript_proto, but it looks to be pretty much the same impl and only a few months older (maybe labs is based off this)

Reply:
fwiw angular-cli is using the labs proto library

Reply:
CC @alexeagle what do you think we should do here?

Reply:
Hi! I don't know the most about this, I've been coasting for a couple years on what Dan did there in labs.

It would be nice to get any JS support from the Goog proto team of course. Assuming they're still first-party only that won't help.
I have seen capable contributions from improbable-eng you might want to look into. And @pcj has done more recent proto work.

Reply:
@evmar if we depend only on the proto library, we don't get TS sources -- how would we construct the proto definition in this case?

Reply:
I was suggesting if we used the protobuf JS library directly, we could consume it from TS by just typing it as 'any'.

I guess this is fine though.  I am worried it will randomly break a few months from now but I guess worst case we can comment it out and revisit if so.